### PR TITLE
Changes for AC-187

### PR DESCRIPTION
**Pull request by red-ship-it**

### CODE REVIEW

File: examples/illustrations/Oresteia/Oresteia.json [hunk 1]
```diff
@@ -656,8 +660,15 @@
                     ],
                     "uco-observable:keypadUnlockCode": "123789",
                     "uco-observable:IMEI": "359305065690067",
-                    "uco-observable:MSISDN": "[clytemnestra-mobileaccount-uuid]",
-                    "uco-observable:clockSetting": "2017-06-21T06:36:24.35Z",
+                    "uco-observable:MSISDN": [
+                        {
+                            "@id": "kb:clytemnestra-mobileaccount-uuid"
+                        }
+                    ],
```
Comment (red-ship-it) [inline examples/illustrations/Oresteia/Oresteia.json:667]: The above is in a json array but I'm not sure if in this case this is OK as I have noted a usage of this that is the similar from the accounts example (https://github.com/casework/CASE-Examples/blob/b0d6daf461b63c6e23d7439ea41050ca60017dd3/examples/illustrations/accounts/accounts.json#L36-L40) 

Also this is what is outlined as the usage in the slides I am working off.

**Comment by ajnelson-nist** [comment]
@red-ship-it , can you merge in the `master` branch, resolving conflicts, and push the update?  I can review the PR then.

File: examples/illustrations/Oresteia/Oresteia.json [hunk 2]
```diff
@@ -661,11 +661,13 @@
...
                     "uco-observable:MSISDN": [
-                        { "kb:clytemnestra-mobileaccount-uuid" }
+                        {
+                            "@id": "kb:clytemnestra-mobileaccount-uuid"
+                        }
                     ],
                     "uco-observable:clockSetting": {
                         "@type": "xsd:dateTime",
-                        "@value": "2017-06-21T06:36:24.35Z"
+                        "@value": " 2017-06-21T06:36:24.35Z"
```
Comment (ajnelson-nist) [inline examples/illustrations/Oresteia/Oresteia.json:670]: A typo has been introduced.  Please remove this whitespace.

**Review by ajnelson-nist — CHANGES_REQUESTED** [review]

File: examples/illustrations/file/file.json [hunk 3]
```diff
@@ -506,15 +506,16 @@
             ]
         },
         {
-            "@id": "kb:android_device1",
+            "@id": "kb:android_device1-uuid",
```
[the remark this answers is not in the document]
Comment (red-ship-it) [reply examples/illustrations/file/file.json:509]: Bookmarking thanks

File: examples/illustrations/Oresteia/Oresteia.json (hunk 2 above)
Comment (ajnelson-nist) [reply examples/illustrations/Oresteia/Oresteia.json]: Resolved in e7e74e9.  Thanks!

**Review by ajnelson-nist — APPROVED** [review]

**Pull request merged**